### ISSUE TITLE
Linter unnecessarily suggests to use test starter when there's more than one script tag in an HTML page

### ISSUE THREAD

**Issue by codeworrior** [UI5/linter#972]
## Expected Behavior

The UI5 linter should not suggest to use the test starter when it is already used (no avoidable false positives)

## Current Behavior

The UI5 linter suggests to use the test starter when an HTML page contains multiple script tags (test starter's `runTest.js` being one of them)

## Steps to Reproduce the Issue

1. Run the UI5 linter on the `sap.m` library and observe the findings for [ResponsiveMarginCssClasses.qunit.html](https://github.com/UI5/openui5/blob/master/src/sap.m/test/sap/m/qunit/ResponsiveMarginCssClasses.qunit.html#L7)

## Context

- UI5 linter version: **`1.20.5`**

## Log Output / Stack Trace

<img width="1037" height="67" alt="Image" src="https://github.com/user-attachments/assets/cfa6a4de-dfe5-4720-aa04-85553d5926ba" />

The problem is most likely caused by the following logic [in the `detectTestStarter` method (lines 64-82)](https://github.com/UI5/linter/blob/main/src/linter/html/transpiler.ts#L64-L82):
```ts
scriptTags.find((tag) => {
		return (... !isTestsuiteQunitFile && <<src attribute is not "resources/sap/ui/test/starter/runTest.js">>);
	});
```
It basically means: "_If there is **any script** that **is not** the test starter's `runScript.js`, then complain_". 
Shouldn't this rather be "_complain if **no script is** the test starter's `runScript.js`_"?
